api-server/src/main/java/io/enmasse/api/server/ApiServer.java

```diff
@@ -76,10 +78,14 @@ public void start(Future<Void> startPromise) throws Exception {
         String requestHeaderClientCa = null;
         try {
             ConfigMap extensionApiserverAuthentication = client.configMaps().inNamespace(options.getApiserverClientCaConfigNamespace()).withName(options.getApiserverClientCaConfigName()).get();
-            clientCa = extensionApiserverAuthentication.getData().get("client-ca.file");
+            clientCa = extensionApiserverAuthentication.getData().get("client-ca-file");
             requestHeaderClientCa = extensionApiserverAuthentication.getData().get("requestheader-client-ca-file");
-        } catch (KubernetesClientException e) {
+            validateCert(clientCa);
+            validateCert(requestHeaderClientCa);
+        } catch (Exception e) {
             log.info("Unable to retrieve config for client CA. Skipping", e);
+            clientCa = null;
```

Comment:
Would it be a good idea to include whether clientCa or requestHeaderClientCa had failed?

Reply:
It is unlikely that one of them fail, but I will add that.